docs/pages/versions/unversioned/sdk/imagepicker.md

```diff
@@ -117,3 +129,30 @@ When you run this example and pick an image, you will see the image that you pic
   "uri":"file:///data/user/0/host.exp.exponent/cache/cropped1814158652.jpg"
 }
 ```
+
+## Constants
+
+### `ImagePicker.MediaTypeOptions`
+
+| Media type                | Accept asset types | Platforms |
+| ------------------------- | ------------------ | --------- |
+| `MediaTypeOptions.All`    | Images and videos  | iOS       |
+| `MediaTypeOptions.Images` | Only images        | both      |
+| `MediaTypeOptions.Videos` | Only videos        | both      |
+
+### `ImagePicker.ExportPresets`
+
+| Preset                         | Resolution            | Video compression algorithm | Audio compression algorithm |
+| ------------------------------ | --------------------- | --------------------------- | --------------------------- |
+| `ExportPresets.Passthrough`    | Unchanged             | None                        | None                        |
```

Comment:
It might be a good idea to also document what are the number values of each of those enum values.